packages/wdio-crossbrowsertesting-service/src/@types/crossbrowsertesting-service.d.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference types="webdriver"/>
```

Comment:
We don't need this file if we keep `packages/wdio-crossbrowsertesting-service/crossbrowsertesting-service.d.ts`

Reply:
Deleted it in the latest commit